### PR TITLE
Naming cleanup

### PR DESCRIPTION
**Pull request by GitGubM8**
# Depends on: 
https://github.com/Thrillberg/imperial/pull/689
[MinDiff](https://github.com/GitGubM8/imperial/compare/main...GitGubM8:imperial:Naming_Cleanup)

## Changes:
- Mostly just cleaning up namings and standardizing display values

### CODE REVIEW

**Comment by GitGubM8** [comment]
I think most of the commits here are from previous PRs, but has since been squashed and merged so git treats them differently

The diff shows a much lighter picture

File: app/javascript/src/stringify.js [hunk 1]
```diff
@@ -1,4 +1,47 @@
-export default (nation) => {
+export const capitalize = (word) => {
```
Comment (Thrillberg) [inline app/javascript/src/stringify.js:1]: I like these changes to `stringify`!

File: app/javascript/src/components/Rondel.vue [hunk 2]
```diff
@@ -196,14 +214,14 @@ export default {
             if (this.game.variant === 'withoutInvestorCard') {
               this.helperText = 'Nation pays players interest';
             } else {
-              this.helperText = 'Nation pays players interest, investor card holder receives 2m and may purchase a bond, Swiss Banks may invest.';
+              this.helperText = `Nation pays players interest, investor card holder receives ${this.displayMonetaryValue_InMillions(2)} and may purchase a bond, Swiss Banks may invest.`;
```
Comment (Thrillberg) [inline app/javascript/src/components/Rondel.vue:208]: Love it! 🤩

**Review by Thrillberg — APPROVED** [review]
Looks great! Thank you @GitGubM8. Feel free to merge when ready!!

File: app/javascript/src/components/Rondel.vue [hunk 3]
```diff
@@ -148,7 +148,19 @@ export default {
...
+    playerBonus() {
+      const taxRevenue = this.game.taxRevenueOf(this.game.currentNation);
+      return this.game.playerBonusAfterUnitMaintenanceCosts(this.game.currentNation, taxRevenue);
+    },
+    nationProfit() {
+      const taxRevenue = this.game.taxRevenueOf(this.game.currentNation);
+      return this.game.nationTaxationProfit(this.game.currentNation, taxRevenue);
+    },
+    nextTaxationPowerPoints() {
+      return nextTaxationPowerPoints(this.game, this.game.currentNation);
```
Comment (GitGubM8) [inline app/javascript/src/components/Rondel.vue:163]: Hold on I think there's a merge conflict here that resulted in duplicated defs

Comment (GitGubM8) [reply app/javascript/src/components/Rondel.vue:163]: Fixed and tested in:
[7e74dee](https://github.com/Thrillberg/imperial/pull/690/commits/7e74dee9889eaa9036539725d3b20670d3b70685)

Comment (Thrillberg) [reply app/javascript/src/components/Rondel.vue:163]: @GitGubM8 Thanks for catching that! 👍

**Pull request merged**